addons/html_editor/static/src/core/format_plugin.js

```diff
@@ -190,14 +190,16 @@ export class FormatPlugin extends Plugin {
      * @returns {boolean}
      */
     isSelectionFormat(format, traversedNodes = this.dependencies.selection.getTraversedNodes()) {
-        const selectedNodes = traversedNodes.filter(isTextNode);
+        const selectedNodes = traversedNodes.filter(
+            (node) =>
+                isTextNode(node) &&
+                !isZwnbsp(node) &&
+                !isEmptyTextNode(node) &&
+                (!/^\n+$/.test(node.nodeValue) || !isBlock(closestElement(node)))
+        );
```

Comment:
This condition tests different stuff than the same commit does in web_editor, in particular it adds a test on `isBlock`. Why is it needed here and not in `web_editor`?